pkg/operator/observe_config.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"gopkg.in/yaml.v2"
```

Comment:
imports go

1. core library
2. not core, not kube, not openshift
3. kube
4. openshift.

Reply:
1. I've notice use of `gopkg.in/yaml.v2` and `github.com/ghodss/yaml` in the `openshift/*` repositories. Any guidance  on when to pick one over another?